app/views/user/show.html.erb

```diff
@@ -0,0 +1,37 @@
+  
+<div class="card mb-3">
+  <div class="row g-0">
+    <div class="col-md-4">
+      <%= link_to user_path(@user) do %>
+        <img src="<%= @user.photo %>" class="img-fluid img-thumbnail" alt="User Profile">
+      <% end %>
+    </div>
+    <div class="col-md-8">
+      <div class="card-body">
+        <%= link_to user_path(@user.id) do %>
+          <h5 class="card-title"><%= @user.name %></h5>
+        <% end %>
+        <p class="card-text"><small class="text-muted">Number of posts: <%= @user.posts.count %></small></p>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="card">
+  <div class="card-body">
+    <h6 class="card-title">Bio</h6>
+    <p class="card-text"><%= @user.bio %></p>
+  </div>
+</div>
+<% @user.posts.each.with_index(1) do |post, index| %>
```

Comment:
- [ ] As the previous reviewer required, please use the method on your user model which is last_three_posts. You can call @recent_posts  that you created in your controller here. Kindly fix this.